website_react/src/components/inputs/InputBoolean.jsx

```diff
@@ -0,0 +1,32 @@
+const InputBoolean = (props) => {
+    return (
+        <div className="form-input-div" key={props.index}>
+            <label className="input-label">{props.formFields.title}</label>
+            <div>
+                <input
```

Comment:
are these two inputs are grouped?

Reply:
yes, as we will have to show both of them everytime for yes/no inputs